mardi_importer/mardi_importer/openml/new_entities.json

```diff
@@ -0,0 +1,108 @@
+{
+    "properties": [
+        {
+            "label": "dataset version",
+            "description": "Version of a dataset",
+            "datatype": "string"
+        },
+        {
+            "label": "collection date",
+            "description": "date as a string",
+            "datatype": "string"
+        },
+        {
+            "label": "upload date",
+            "description": "upload date of file",
+            "datatype": "time"
+        },
+        {
+            "label": "default target attribute",
+            "description": "the default target attribute",
+            "datatype": "string"
+        },
+        {
+            "label": "row id attribute",
+            "description": "the row id attribute",
+            "datatype": "string"
+        },
+        {
+            "label": "OpenML semantic tag",
+            "description": "the OpenML semantic tag",
+            "datatype": "string"
+        },
+        {
+            "label": "has feature",
+            "description": "it has feature",
+            "datatype": "string"
+        },
+        {
+            "label": "data type",
+            "description": "the data type",
+            "datatype": "string"
+        },
+        {
+            "label": "number of binary features",
+            "description": "the number of binary features",
+            "datatype": "quantity"
+        },
+        {
+            "label": "number of classes",
+            "description": "the number of classes",
+            "datatype": "quantity"
+        },
+        {
+            "label": "author name string",
```

Comment:
I think it is not necessary to add this, is it? We already have imported this from wikidata: https://portal.mardi4nfdi.de/wiki/Property:P43